src/libraries/Microsoft.PowerFx.Interpreter/SymbolContext.cs

```diff
@@ -62,6 +62,11 @@ public SymbolContext WithScopeValues(ErrorValue scopeValues)
             return WithScopeValues(new ErrorScope(scopeValues));
         }
 
+        public SymbolContext WithScopeValues(BlankValue value)
+        {
+            return WithScopeValues(new BlankScope(value));
+        }
```

Comment:
What about: 
public SymbolContext WithScopeValues(FormulaValue  scopeValues)

Or 
public SymbolContext WithScopeValues(DValue<RecordValue> scopeValues)


which does the switch and calls the right type. 
Else, I'm noticing the caller has to do the switch, and often it is just pass through